CRUDRecipeEF.BL.DL/Services/IngredientService.cs

```diff
@@ -70,6 +70,19 @@ public async Task<string> AddIngredient(IngredientAddDTO ingredientAddDTO)
             return ingredientAddDTO.Name;
         }
 
+        public async Task UpdateIngredient(string name, IngredientAddDTO ingredientAddDTO)
```

Comment:
Ingredient services currently has this update method https://github.com/chrisK00/CRUDRecipeEF/blob/master/CRUDRecipeEF.BL.DL/Services/IIngredientService.cs